src/NuGet.Clients/NuGet.VisualStudio.Internal.Contracts/INuGetProjectManagerService.cs

```diff
@@ -18,6 +19,12 @@ public interface INuGetProjectManagerService : IDisposable
         ValueTask<IReadOnlyCollection<IPackageReferenceContextInfo>> GetInstalledPackagesAsync(
             IReadOnlyCollection<string> projectIds,
             CancellationToken cancellationToken);
+        ValueTask<(IReadOnlyCollection<IPackageReferenceContextInfo>, IReadOnlyCollection<IPackageReferenceContextInfo>)> GetAllPackagesAsync(
```

Comment:
cc @rrelyea @dtivel @sbanni 

Can one of you take a look at these codespaces APIs

Reply:
I look at this...

Reply:
I added the interface IInstalledAndTransitivePackages and a formatter.